src/timer.h

```diff
@@ -62,54 +54,50 @@ typedef struct timespec hr_timer;
 #define HRZERO(X)             (X).tv_sec = (X).tv_nsec = 0
 #define HRSETCURRENT(X)       clock_gettime(BEST_MONOTONIC, &(X))
 #define HRGETTICKS(X)         ((uint64_t)(X).tv_sec * 1000000000ULL + (uint64_t)(X).tv_nsec)
-#define HRGETTICKS_PER_SEC(X) { hr_timer r; clock_getres(BEST_MONOTONIC, &r); if (r.tv_sec < 0) clock_getres(CLOCK_MONOTONIC, &r); (X) = r.tv_sec + 1000000000ULL / r.tv_nsec; }
+#define HRGETTICKS_PER_SEC(X) { hr_timer r; clock_getres(BEST_MONOTONIC, &r); if (r.tv_sec < 0) clock_getres(CLOCK_MONOTONIC, &r); (X) = r.tv_sec + 1000000000ULL * r.tv_nsec; }
```

Comment:
Why are you changing `/` to `*` here? Both expressions look wrong to me. Do we use this macro anywhere? I hope not.

Reply:
Haven't found a hard spec yet but actual values I've ever seen is `sec` as 0 and `nsec` as 1 when result should be 1,000,000,000 and in that case it doesn't matter but I have a feeling we should just use `tv_nsec` as-is and ignore `tv_sec`.

Reply:
I was confused by `tv_sec` being there at all. Looking at it again, it should likely be `(X) = 1000000000ULL / r.tv_nsec;`. Then if we get `nsec` as 1000, we'll get 1,000,000 ticks per second.

Reply:
Right, but I hope we can simply drop the `HRGETTICKS_PER_SEC` macro. I doubt we have a use for it, and if we do I'd question that use first.

Reply:
Oh, right. I never realized all macros and functions are using (the result of) it needed or not. Perhaps the Windows macros actually need it though?

Reply:
Oh, and macOS do too. Things can be made better though.